From 121949c7fb0729d36e2c378171aecda99912a4c9 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Mon, 27 Jun 2022 18:45:02 +0200
Subject: [PATCH 06/11] arm64: dts: rockchip: rk3399-pinebook-pro: Support both
 Type-C plug orientations

This fixes USB 3.0 and Alt-DP modes in reverse plug orientation.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
index 8080247f6e5e..462f5dd08d60 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
@@ -1118,6 +1118,7 @@ &usbdrd3_0 {
 &usbdrd_dwc3_0 {
 	dr_mode = "otg";
 	extcon = <&typec_extcon_bridge>;
+	snps,usb3-phy-reset-quirk;
 	status = "okay";
 };
 
-- 
2.43.0

